plugins/inputs/t128_tank/README.md

```diff
@@ -0,0 +1,30 @@
+# 128T TANK Input Plugin
+
+The tank input plugin collects data from a 128T.
```

Comment:
@gregschrock can you recommend a better blurb here.

Reply:
```
The `t128_tank` input plugin reads from a [TANK](https://github.com/phaistos-networks/TANK) queue and creates metrics using one of the supported [input data formats](/docs/DATA_FORMATS_INPUT.md).
```